packages/mgt/src/graph/graph.people.ts

```diff
@@ -160,6 +160,49 @@ export async function findPeople(
   return graphResult ? graphResult.value : null;
 }
 
+/**
+ * async promise, returns all matching Graph users who are member of the specified group
+ *
+ * @param {string} query
+ * @param {string} groupId - the group to query
+ * @param {number} [top=10] - number of people to return
+ * @param {PersonType} [personType=PersonType.person] - the type of person to search for
+ * @returns {(Promise<Person[]>)}
+ */
+export async function findPeopleFromGroup(
+  graph: IGraph,
+  query: string,
+  groupId: string,
+  top: number = 10,
+  personType: PersonType = PersonType.person
+): Promise<Person[]> {
+  const scopes = 'user.read.all';
+
+  let cache: CacheStore<CacheGroupPeople>;
+
+  if (peopleCacheEnabled()) {
+    cache = CacheService.getCache<CacheGroupPeople>(cacheSchema, groupStore);
+    const peopleItem = await cache.getValue(`${query}:${groupId}`);
+    if (peopleItem && getPeopleInvalidationTime() > Date.now() - peopleItem.timeCached) {
+      return peopleItem.people.map(peopleStr => JSON.parse(peopleStr));
+    }
+  }
+
+  const graphResult = await graph
+    .api(`/groups/${groupId}/members`)
```

Comment:
Just a thought - should we use [`transitiveMembers`](https://docs.microsoft.com/en-us/graph/api/group-list-transitivemembers?view=graph-rest-1.0&tabs=http) instead of `members`?

Reply:
I was thinking about it... I don't hate the idea, just wondering if there is a use case for just `members` in this case then. Should this be a config on the component?

Reply:
Great idea - are you thinking about a new property? How about a boolean property/attribute like `transitiveSearch` or `recursiveSearch` that is used when group-id is set?

Reply:
Exactly what I was thinking. `transitiveSearch` would be closer to the Graph terminology, so that is what I would use!

Reply:
Awesome, do you want to do it here, or should we create a separate issue for it? Either one works for me.

Reply:
I'll make it happen here, I'm already on the case anyway!